cypress/integration/login.js

```diff
@@ -12,16 +12,17 @@ describe('Sourcebooks login', function() {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Jevgenija Malomuž"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
         cy.get('[aria-label="User"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
+        cy.get('.calendar--today').contains(new Date().getDate());

```

Comment:
Please move this line after line: 
 cy.get('.calendar').should('be.visible')
cypress is flexible and not failing in this case but other frameworks are not waiting for element to be visible